tasks/letsencrypt/letsencrypt.rake

```diff
@@ -0,0 +1,42 @@
+require_relative 'lets_encrypt_route53'
+
+namespace :letsencrypt do
+  le = LetsEncryptRoute53.new.tap do |config|
+    config.endpoint           = LetsEncryptRoute53::PRODUCTION
+    config.domains            = ['staging.oversight.garden',
+                                 'www.staging.oversight.garden']
+    config.region             = 'us-east-1'
+    config.s3_bucket          = 'oversight-secrets'
+    config.s3_key_acct        = 'letsencrypt.acct.pem'
+    config.s3_key_privkey     = 'letsencrypt.privkey.pem'
+    config.s3_key_cert        = 'letsencrypt.cert.pem'
+    config.s3_key_chain       = 'letsencrypt.chain.pem'
+    config.s3_key_fullchain   = 'letsencrypt.fullchain.pem'
+    config.path_acct          = '/home/ubuntu/letsencrypt.acct.pem'
+    config.path_privkey       = '/home/ubuntu/letsencrypt.privkey.pem'
+    config.path_cert          = '/home/ubuntu/letsencrypt.cert.pem'
+    config.path_chain         = '/home/ubuntu/letsencrypt.chain.pem'
+    config.path_fullchain     = '/home/ubuntu/letsencrypt.fullchain.pem'
+    config.kms_key_id         = '66c6c116-e0f7-425e-b2f6-a750373a242e'
+    config.contact_email      = 'eric@konklone.com'
```

Comment:
`eric+oversight@konklone.com` is slightly preferred for an interim, but obviously the right solution is to have an email address that's `@oversight.garden` -- which I'm up for making any time.